pkg/user/delivery/http/authn_proxy.go

```diff
@@ -10,52 +10,67 @@ import (
 	"github.com/CESARBR/knot-babeltower/pkg/user/entities"
 )
 
-// authn documentation: https://github.com/mainflux/mainflux/blob/master/authn/swagger.yaml
-
-// AuthnProxy represents the interface to the authn's proxy operations
+// AuthnProxy is an interface to the Mainflux Authn service, which provides an API for.
+// managing authentication keys. This interface provides a way to create application
+// tokens (with configurable duration) and to validate the token used in operations across
+// the platform.
+// https://github.com/mainflux/mainflux/blob/0.11.0/authn/swagger.yaml
 type AuthnProxy interface {
 	CreateAppToken(user entities.User, duration int) (token string, err error)
+	IsValidToken(email, token string) bool
```

Comment:
This is being added before it's creation (or github is messing the commit order, again LOL) :eyes:

Reply:
Not ready for review yet. :)

Reply:
Done.